libs/model-ad/api-client-angular/src/lib/.openapi-generator/FILES

```diff
@@ -14,6 +14,7 @@ model/basicError.ts
 model/comparisonToolConfig.ts
 model/comparisonToolConfigColumn.ts
 model/comparisonToolConfigFilter.ts
+model/comparisonToolPage.ts
```

Comment:
it looks like the OpenAPI generator creates files using camel case while we usually prefer kebab case. This leads to imports with mixed casing, which we may want to address in the future.